lib/gui/utils/byte-size/byte-size.js

```diff
@@ -26,6 +26,11 @@
 const angular = require('angular');
 const MODULE_NAME = 'Etcher.Utils.ByteSize';
 const ByteSize = angular.module(MODULE_NAME, []);
+
+/* eslint-disable lodash/prefer-lodash-method */
```

Comment:
Would it make sense to loosen up the eslint rule, instead of having to pepper the code with these comments (since the number of exceptions is quite high with this one)?